test/usage.cjs

```diff
@@ -2602,9 +2602,10 @@ describe('usage tests', () => {
       r.logs[0]
         .split('\n')
         .should.deep.equal([
-          'usage',
+          'usage [command]',
           '',
-          'do something',
+          'Commands:',
```

Comment:
It's by design that we don't show the list of commands unless there is at least one command registered that's not the default command, this is because all the information the user needs can be expressed like this:

```
example.mjs

the default command

Options:
  --help     Show help                                                 [boolean]
  --version  Show version number                                       [boolean]
```

I don't think this is the bug you were trying to fix though?

Reply:
I'm guessing your fix addresses another but you were having? but I think we need a test for it, and this test shouldn't change.